packages/react-components/src/organisms/TeamCreateOutputHeader.tsx

```diff
@@ -21,18 +21,97 @@ const visualHeaderStyles = css({
   maxWidth: `${720 / perRem}em`,
 });
 
+const headerCopy = ({ outputType }: { outputType: ResearchOutput['type'] }) => {
+  switch (outputType) {
+    case 'Protocol':
+      return 'Share a protocol';
+    case 'Dataset':
+      return 'Share a dataset';
+    case 'Bioinformatics':
+      return 'Share bioinformatics';
+    case 'Lab Resource':
+      return 'Share a lab resource';
+    case 'Article':
+      return 'Share an article';
+    default:
+      return 'Share a resource';
+  }
+};
+
+const SubheaderCopy: React.FC<{ outputType: ResearchOutput['type'] }> = ({
+  outputType,
+}) => {
+  switch (outputType) {
+    case 'Protocol':
+      return (
+        <>
+          Add your protocol to a repository (e.g. protocols.io) before sharing
+          on the Hub. Find out{' '}
+          <Link
+            href={
+              'https://hub.asap.science/news/7d3dd1ec-14ef-441c-8a2c-19a23d6264f3'
```

Comment:
Mhm... the content only exists on production, using the routing library would mean on dev/PR envs it points to nothing. I'd go with this because we are unlikely to change the domain/routes - and then we only have two broken links.

But I have no hard feelings about this...

Reply:
This link exists in development https://dev.hub.asap.science/news/7d3dd1ec-14ef-441c-8a2c-19a23d6264f3

Reply:
Oh, I didn't expect that. Will change 👍